plugins/web/README.md

```diff
@@ -2,12 +2,14 @@
 
 This plugin provides a web server and access to the REST frontend for VAST.
 
+The following describes the backend web server part of the web plugin. For information regarding the frontend vast UI, see [this](./ui/README.md).
```

Comment:
```suggestion
The following describes the backend web server part of the web plugin. For information regarding the frontend VAST UI, see [this](./ui/README.md).
```

In code or slugs, use `vast`, but when talking about the product, use VAST as the productized spelling.

Nit: Please limit this to 80 cols per the style guide.